src/Controller/OcrController.php

```diff
@@ -20,11 +19,21 @@ class OcrController extends AbstractController
     /** @var Intuition */
     protected $intuition;
 
-    /** @var EngineBase */
+    /** @var TesseractEngine|GoogleCloudVisionEngine */
     protected $engine;
 
-    /** @var mixed[] Output params for the view or API response. */
-    protected $params = [];
+    /**
+     * The output params for the view or API response.
+     * This also serves as where you define the defaults.
+     * Note this is used by the ExceptionListener.
+     * @var mixed[]
+     */
+    public static $params = [
```

Comment:
Should we look at adding the `dpi` option? We currently get warnings on most images:

> Warning: Invalid resolution 0 dpi. Using 70 instead.

I'm happy to hold that over for a future patch, but will it work with this system? It won't have a max value (well, I guess it will in practice, but it's a different sort of value).

Reply:
From playing around, it seems there is a valid DPI range from 70 to 2400. I added a DPI option, but depending on what other options you use, you still get the same error. For instance, with a PSM of 0, and OEM of 3, and DPI of 300, I see:
```
Error! The command did not produce any output.

Generated command:
"tesseract" - "/tmp/ocrXMeZIy" --psm 0 --oem 3 --dpi 300

Returned message:
Tesseract Open Source OCR Engine v4.1.1 with Leptonica
Warning. Invalid resolution 0 dpi. Using 70 instead.
```

Seems like there might be an issue with thiagoalessio/tesseract_ocr. But either way, I have no idea what options require you to set a DPI, and I suspect many of our users won't, either. Do we know what the conditions are where DPI is needed?

Reply:
Hmm yeah good points. I think the DPI is probably most useful for things like newspapers, which might also have a scan resolution set in the Index page. Let's leave it for now.